cmake/module/FindQt.cmake

```diff
@@ -40,10 +40,10 @@ endif()
 # /usr/x86_64-w64-mingw32/lib/libm.a or /usr/arm-linux-gnueabihf/lib/libm.a.
 set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY BOTH)
 
-find_package(Qt5 ${Qt5_FIND_VERSION}
-  COMPONENTS ${Qt5_FIND_COMPONENTS}
+find_package(Qt${Qt_FIND_VERSION_MAJOR} ${Qt_FIND_VERSION}
+  COMPONENTS ${Qt_FIND_COMPONENTS}
```

Comment:
nit: is there a reason for keeping the `t` lowercase? Looks weird this way...

Reply:
Variables such as `<PackageName>_FIND_VERSION`, `<PackageName>_FIND_VERSION_MAJOR`, and `<PackageName>_FIND_COMPONENTS`, are part of the `Find<PackageName>` [interface](https://cmake.org/cmake/help/latest/command/find_package.html#package-file-interface-variables). In our case, `<PackageName>` is replaced with `Qt`.

Reply:
Wow, so muc magic